qiskit/mapper/_mapping.py

```diff
@@ -590,9 +590,9 @@ def swap_mapper(circuit_graph, coupling_graph,
             dagcircuit_output.compose_back(layer["graph"], layout)
 
     # Parse openqasm_output into DAGCircuit object
-    dag_unrrolled = DagUnroller(dagcircuit_output,
+    dag_unrolled = DagUnroller(dagcircuit_output,
                                 DAGBackend(basis.split(",")))
```

Comment:
make style triggers a "over-indented error" (see https://travis-ci.org/Qiskit/qiskit-terra/jobs/446757731#L601) because the line 593 is one char shorter now. Here is the fix:

```suggestion
dag_unrolled = DagUnroller(dagcircuit_output,
                           DAGBackend(basis.split(",")))
```